common.props

```diff
@@ -4,7 +4,7 @@
 
     <MajorProductVersion>4</MajorProductVersion>
     <MinorProductVersion>0</MinorProductVersion>
-    <PatchProductVersion>1</PatchProductVersion>
+    <PatchProductVersion>2</PatchProductVersion>
```

Comment:
I published the nuget package locally and consumed it in an in-proc app and verified that it works fine (and all bindings are used in the binding metric logging code path)